src/server/install/index.md

```diff
@@ -18,6 +18,8 @@ The <CommunityPlatformName /> images are stored on publicly accessible <DockerHu
 
 Afterwards, you can follow [this guide](./ee/index.md) to retrieve your <EnterprisePlatformName /> images.
 
+Make sure you follow deployment guidelines, namely enabling [`call-home`](../administer/index.md#opt-out-of-sending-statistics) functionality.
+
```

Comment:
This part could be clearer - I don't like the enabling -> opt out - I fear it could be misinterpreted as people being told they should opt-out. 

As this part is in the EE install guide, I would instead turn it into an instruction for them to *ensure any firewalls in your infrastructure are configured to allow the call-home functionality to send usage data*